Kernel/VirtIO/VirtIO.h

```diff
@@ -196,7 +195,8 @@ class VirtIODevice : public PCI::Device {
         return is_feature_set(m_accepted_features, feature);
     }
 
-    void supply_buffer_and_notify(u16 queue_index, const ScatterGatherList&, BufferType, void* token);
+    void supply_chain_and_notify(u16 queue_index, VirtIOQueueChain& chain);
+    void notify_queue(u16 queue_index);
```

Comment:
how about keeping it private now that the helper method supply_chain_and_notify is available?

Reply:
Good idea, will do in next revision